### DISCUSSION TITLE
关于easyPSI项目进行隐私求交失败

### DISCUSSION THREAD

**Discussion by akeykid** [secretflow/secretflow#1223] [Q&A]
任务启动后无法成功，报错信息如下：

proot error: ptrace(TRACEME): Operation not permitted
proot error: execve("/usr/bin/sh"): Operation not permitted
proot info: It seems your kernel contains this bug: https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1202161
To workaround it, set the env. variable PROOT_NO_SECCOMP to 1.
fatal error: see `proot --help`.
proot error: can't chmod '/tmp/proot-1628-ssVgSd': No such file or directory
![微信图片_20240401195144](https://github.com/secretflow/secretflow/assets/6858495/4fbf40dc-ba50-4128-ae5f-978c91bacbb9)

**Comment by 6fj** [comment]
hi @akeykid 

根据 https://www.secretflow.org.cn/zh-CN/docs/easy-psi/latest/wl2hba8vbkgpea4i#pBMyH，
如果你使用docker方式部署，可能需要：
- 增加 --cap-add=SYS_PTRACE参数
- 添加环境变量参数 -e PROOT_NO_SECCOMP=1

后续问题，烦请在https://github.com/secretflow/easy-psi/issues 提问，感谢！
